### ISSUE TITLE
Can't move Hydrogen Cell into Chemical Reactor

### ISSUE THREAD

**Issue by Eared** [GTNewHorizons/GT-New-Horizons-Modpack#7110]
#### Which modpack version are you using?
2.0.9.0

#### If in multiplayer; On which server does this happen? 
My local server

#### What did you try to do, and what did you expect to happen?
When I try to push Hydrogen Cell into Chemical Reactor using Item Conduits it doesn't happens
(the same for project red pipes&chip or just trying to get item with conveyor on machine from chest)
Same for Steam Cells and whatever Chemical Reactor can accept I believe
Also apparently if Chemical Reactor have cell already inside it will take input as discovered by Alsarus on discord

#### What happened instead? (Attach screenshots if needed)
![image](https://user-images.githubusercontent.com/38147939/102273728-9bd9cb80-3f44-11eb-85b2-8bd2e59164e4.png)
![image](https://user-images.githubusercontent.com/38147939/102273793-b0b65f00-3f44-11eb-8c05-c58a0b6702a9.png)
![image](https://user-images.githubusercontent.com/38147939/102273805-b57b1300-3f44-11eb-9243-c46c6da468b0.png)
![image](https://user-images.githubusercontent.com/38147939/102273812-b9a73080-3f44-11eb-8da2-c9a55fdb99fb.png)
![image](https://user-images.githubusercontent.com/38147939/102340466-640c6b80-3fb8-11eb-8db3-730acba0eb93.png)

Items got into pipe but weren't send inside since it had chip in input inventory
![image](https://user-images.githubusercontent.com/38147939/102274526-dc861480-3f45-11eb-8dfb-3139a11026e8.png)

upd: it will take Hydrogen cells but only when powered which was confusing

#### What do you suggest instead/what changes do you propose?
tldr: won't take cells in unless it's powered or had same item already inside
fix it?

**Comment by boubou19** [comment]
Looks similar to https://github.com/GTNewHorizons/GT-New-Horizons-Modpack/issues/7179
